force-app/main/default/classes/WiredActsController.cls

```diff
@@ -11,14 +11,14 @@ public with sharing class WiredActsController {
        list<account> acts= [Select id,name,phone from account where name like :n];
        system.debug(acts); 
        return acts;      
-    } 
```

Comment:
why there is a space

Reply:
it came like that when i checked out that code from git